libs/blocks/modal/modal.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-underscore-dangle */
 /* eslint-disable import/no-cycle */
 import { createTag, getMetadata, localizeLink, loadStyle, getConfig } from '../../utils/utils.js';
+import { getFederatedUrl } from '../../utils/federated.js';
```

Comment:
I'd conditionally import this just before ` const block = createTag('a', { href: getFederatedUrl(path) });` only IF an URL includes 'federal' - I imagine it's a quite niche use-case for the modal, thus we don't want to have this import for every single modal use-case

Reply:
Added the conditional import.